tests/Tests/400__ods_dashboard/412__ods_dashboard_resources.robot

```diff
@@ -119,3 +126,53 @@ Verify Quick Starts Work As Expected When At Least One Step Is Skipped
     END
     Run Keyword And Continue On Failure     QuickStart Status Should Be    ${element}      In Progress
     Run Keyword And Continue On Failure     Link Text On QuickStart Card Should Be  element=${element}  exp_link_text=Continue
+
+External URLs Should Not Be Broken
+    [Documentation]     Go through a QuickStart and checks the status of all the external links
+    ${sideWindowButtons}=   Get WebElements   //button[@class='pf-c-wizard__nav-link']
+    ${element_list}=    Get WebElements    xpath=//div[@Class="pf-c-drawer__panel-main"]//a[@href]
+    URLs HTTP Status Code Should Be Equal To    ${element_list}
+
+    FOR    ${sideWindowButton}    IN     @{sideWindowButtons}
```

Comment:
this `sideWindowButton` represent the single steps in the QS, right? Then I'd suggest to call them `quick_start_steps` or similar

Reply:
Done